### PR TITLE
Clarify axis convention language

### PR DESCRIPTION
**Pull request by dubois**
Apply the change suggested in #1640

### CODE REVIEW

File: specification/2.0/README.md [hunk 1]
```diff
@@ -251,7 +251,7 @@ For property names, glTF uses [camel case](http://en.wikipedia.org/wiki/CamelCas
 
 ## Coordinate System and Units
 
-glTF uses a right-handed coordinate system, that is, the cross product of +X and +Y yields +Z. glTF defines +Y as up. The front of a glTF asset faces +Z.
+glTF uses a right-handed coordinate system; that is, the cross product of right and forward yields up. glTF defines +Y as up, +Z as forward, and -X as right.
```
Comment (donmccurdy) [inline specification/2.0/README.md:254]: The line, "The front of a glTF asset faces +Z." is a useful clarification of "+Z as forward", as opposed to forward in some scene navigation or default camera orientation sense. I think I would keep that part. Otherwise looks good, thank you!

Comment (dubois) [reply specification/2.0/README.md]: Done.

Comment (donmccurdy) [reply specification/2.0/README.md]: Had to read this through a couple times:

> ... the cross product of right and forward yields up ...

I think you mean `left ⨉ forward`? It may be better to stick with the original phrasing of this clause, "cross product of +X and +Y yields +Z". The right-hand rule does not really dictate the right/forward/up convention to my understanding (see various combinations here: https://www.evl.uic.edu/ralph/508S98/coordinates.html).

And thanks for correcting the comma. :)

Comment (donmccurdy) [reply specification/2.0/README.md]: On second reading, my link above contradicts us:

> The default coordinate system in OpenGL(TM) is right-handed: the positive x and y axes point right and up, and the negative z axis points forward. 

This might need a closer review, @lexaknyazev would be interested in your thoughts.

Comment (lexaknyazev) [reply specification/2.0/README.md]: +X right and -Z forward is the right-handed system same as +X left and +Z forward, just rotated by 180 degrees.

File: specification/2.0/README.md [hunk 2]
```diff
@@ -251,7 +251,7 @@ For property names, glTF uses [camel case](http://en.wikipedia.org/wiki/CamelCas
 
 ## Coordinate System and Units
 
-glTF uses a right-handed coordinate system, that is, the cross product of +X and +Y yields +Z. glTF defines +Y as up. The front of a glTF asset faces +Z.
+glTF uses a right-handed coordinate system; that is, the cross product of right and forward yields up. glTF defines +Y as up, +Z as forward, and -X as right. The front of a glTF asset faces +Z.
```
Comment (donmccurdy) [inline specification/2.0/README.md:254]:
```suggestion
glTF uses a right-handed coordinate system; that is, the cross product of +X and +Y yields +Z. glTF defines +Y as up, +Z as forward, and -X as right. The front of a glTF asset faces +Z.
```

Comment (lexaknyazev) [reply specification/2.0/README.md:254]: Arguable, the whole "cross product" part is redundant as it's true for any right-handed system.

File: specification/2.0/README.md (hunk 1 above)
Comment (donmccurdy) [reply specification/2.0/README.md]: Ah thanks! I'm misreading things here. 

Sounds like this can be resolved.

File: specification/2.0/README.md (hunk 2 above)
Comment (lexaknyazev) [inline specification/2.0/README.md:254]: While this statement is technically correct, it's hard to parse because the "right" direction is defined after it's used.

Comment (lexaknyazev) [reply specification/2.0/README.md:254]: Removing the "the cross product ..." would eliminate the confusion without affecting normative definitions.

**Comment by donmccurdy** [comment]
GitHub is showing that I deleted a comment from @lexaknyazev — I'm not sure how (I only deleted a "suggested edit" of my own?) but let me know if something has gone missing here.

> Screen Shot 2021-04-14 at 9 56 41 PM

**Comment by lexaknyazev** [comment]
> GitHub is showing that I deleted a comment from @lexaknyazev — I'm not sure how (I only deleted a "suggested edit" of my own?) but let me know if something has gone missing here.

Removing a suggestion apparently removes all comments on it. I [repeated](https://github.com/KhronosGroup/glTF/pull/1969#discussion_r613754019) my comment above.

File: specification/2.0/README.md (hunk 2 above)
Comment (dubois) [reply specification/2.0/README.md]: Missed this comment! Done.

**Comment by emackey** [comment]
Looks like this was manually merged into the jon-adoc branch [here](https://github.com/KhronosGroup/glTF/blob/7748663963f65dba3cd6727a2fb2037f8ca58413/specification/2.0/Specification.adoc#coordinate-system-and-units).

Thanks @dubois, I'm going to close this PR as the old Markdown version is going away soon, but this change is already incorporated in the upcoming version.
